src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/Public.Data/PreviewTokenCreateRequest.cs

```diff
@@ -8,17 +8,61 @@ public record PreviewTokenCreateRequest
     public required Guid DataSetVersionId { get; init; }
 
     public required string Label { get; init; }
-
+    
+    public DateTimeOffset? Activates { get; init; }
+    
+    public DateTimeOffset? Expires { get; init; }
+    
     public class Validator : AbstractValidator<PreviewTokenCreateRequest>
     {
         public Validator()
         {
+            var utcNow = DateTimeOffset.UtcNow;
+
+            RuleFor(request => request.Activates!.Value)
```

Comment:
Can't this just be `RuleFor(request => request.Activates)`?

This comment applies elsewhere.